external/model-preparation-algorithm/configs/classification/configuration.yaml

```diff
@@ -82,7 +82,7 @@ learning_parameters:
     visible_in_ui: true
     warning: null
   num_workers:
-    affects_outcome_of: NONE
+    affects_outcome_of: TRAINING
```

Comment:
Speed changes but the output does not.
@harimkang What do you think?

Reply:
What does this variable do? Does that mean it's only used for training?

Reply:
> Speed changes but the output does not. @harimkang What do you think?

Hmm, I can't understand this sentence. what's the meaning of the output?

Reply:
It seems that 'learning rate' like h-params are marke as 'affects_outcome_of: TRAINING'.

Reply:
Yeah, if we want to override a specific hyper-parameter defined on `configuration` (i.e. `learning rate`), then that variable should be set as `affects_outcome_of = TRAINING or INFERENCE`. 

So, in this case, because I want to override `num_workers` variable, I changed the `affects_outcome_of = TRAINING`. And add additional argument `learning_parameters.num_workers=0` for passing the OTE CI test.

However, as @harimkang said in PR #1367, if there is no problem on `num_workers`, then let's drop this PR